src/lang/terms/triple.pl

```diff
@@ -526,6 +529,27 @@
 %%%%%%%%% triple/3 query pattern
 %%%%%%%%%%%%%%%%%%%%%%%
 
+%% triple(?Subject, ?Property, ?Value) is nondet.
+%
+% Query values of a property on some subject in the triple DB.
+%
+% The property can be wrapped in one of several options:
+% 
+% - transitive/1 indicates that the property is transitive
+% - reflexive/1 indicates that the property is reflexive
+% - pstar/1 binds the property to one of the values in the p* field in the mongodb
```

Comment:
Have you checked how it looks in the HTML page?
In query.pl formatting for options is slightly different:

```
% Options include:
%
%     - max_queue_size(MaxSize)
%     Determines the maximum number of messages queued in each stage.  Default is 50.
```

Then it will look more appealing in the HTML page.

Reply:
`rosrun rosprolog rosprolog-doc knowrob`

Reply:
> Have you checked how it looks in the HTML page?
> In query.pl formatting for options is slightly different:
> 
> ```
> % Options include:
> %
> %     - max_queue_size(MaxSize)
> %     Determines the maximum number of messages queued in each stage.  Default is 50.
> ```
> 
> Then it will look more appealing in the HTML page.

Could be a bit confusing, because this options in query.pl belong to a list of options. In the case of triple the options are more wrapper around the properties/values.